### PR TITLE
release: Bumped to 5.0.0

### PR DESCRIPTION
**Pull request by parkhomenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,12 +169,40 @@ For more information about Fauna Query Language (FQL), consult our query languag
...
+```
+where each wrapper type returned from an error result according to the error code from fauna database server,
+you can inspect "errors.go" file for more information on how it is implemented.
+
+You should now notice from your error handling code that the error message has slight changes,
+and you are now able to use wrapper names to check the error type,  
+take a look at the example of handling an error for non-existent collection  
+that are taken from one of the unit tests (you can see more such examples in "client_test.go"):
+```
+_, err := s.client.Query(
```
Comment (faunaee) [inline README.md:196]: What is `s` here? The example higher up in this file imports `faunadb` into `f`, and then creates a bare `client`. Our examples should be consistent among themselves.

File: README.md [hunk 2]
```diff
@@ -169,12 +169,40 @@ For more information about Fauna Query Language (FQL), consult our query languag
...
+    f.Paginate(f.Documents(f.Collection("spells_not_exists"))),
+)
+
+if _, ok := err.(f.InvalidReferenceError); !ok {
+    s.Require().Fail("Should have returned InvalidReferenceError")
+}
+
+s.EqualError(err, "Response error 400. Errors: [paginate/documents](invalid ref): Ref refers to undefined collection 'spells_not_exists', details: []")
```
Comment (faunaee) [inline README.md:204]: Would a user use `s.EqualError` in their own client applications?

File: README.md [hunk 3]
```diff
@@ -169,12 +169,40 @@ For more information about Fauna Query Language (FQL), consult our query languag
...
+take a look at the example of handling an error for non-existent collection  
+that are taken from one of the unit tests (you can see more such examples in "client_test.go"):
+```
+_, err := s.client.Query(
+    f.Paginate(f.Documents(f.Collection("spells_not_exists"))),
+)
+
+if _, ok := err.(f.InvalidReferenceError); !ok {
+    s.Require().Fail("Should have returned InvalidReferenceError")
+}
```
Comment (faunaee) [inline README.md:202]: How would a use get the `httpStatusCode`, message/description, or other details when an error occurs?

**Review by faunaee — COMMENTED** [review]
I don't think that we're done with the driver consistency effort yet.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
-# v4.0.0 (April, 2021) [current]
+# v5.0.0 (October, 2021) [current]
+
+- Supports fauna database API v5
```
[the remark this answers is not in the document]
Comment (parkhomenko) [reply CHANGELOG.md:3]: we discussed this with @macnealefauna and @breinero that we need to include database v5 API for the current driver release

[the remark this answers is not in the document]
Comment (parkhomenko) [reply CHANGELOG.md:3]: here is the actual code change that moves all API calls to v5:
https://github.com/fauna/faunadb-go/blob/release-5.0.0/faunadb/client.go#L24

File: README.md (hunk 1 above)
Comment (parkhomenko) [reply README.md:196]: fixed

File: README.md (hunk 2 above)
Comment (parkhomenko) [reply README.md:204]: fixed

**Comment by parkhomenko** [comment]
Close in favour of this one:
https://github.com/fauna/faunadb-go/pull/160

**Pull request closed without merging**